.appveyor.yml

```diff
@@ -1,215 +1,143 @@
 image: Visual Studio 2015
 version: '{branch}.{build}'
 
-clone_folder: c:\projects\xdebug
+clone_folder: C:\projects\xdebug
 
 install:
-    ps: |
-        if (-not (Test-Path c:\build-cache)) {
-            mkdir c:\build-cache
-        }
-        $bname = 'php-sdk-' + $env:BIN_SDK_VER + '.zip'
-        if (-not (Test-Path c:\build-cache\$bname)) {
-            Invoke-WebRequest "https://github.com/OSTC/php-sdk-binary-tools/archive/$bname" -OutFile "c:\build-cache\$bname"
-        }
-        $dname0 = 'php-sdk-binary-tools-php-sdk-' + $env:BIN_SDK_VER
-        $dname1 = 'php-sdk-' + $env:BIN_SDK_VER
-        if (-not (Test-Path c:\build-cache\$dname1)) {
-            7z x c:\build-cache\$bname -oc:\build-cache
-            move c:\build-cache\$dname0 c:\build-cache\$dname1
-        }
+    ps: .appveyor\install.ps1
 
 cache:
-  - c:\build-cache -> .appveyor.yml
+  - C:\build-cache -> .appveyor.yml, .appveyor\install.ps1
 
 environment:
     BIN_SDK_VER: 2.2.0
     matrix:
-        - PHP_VER: 7.4.0
+        - PHP_VER: master
+          ARCH: x64
+          TS: 1
+          VC: vs16
+          OPCACHE: yes
+          APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
```

Comment:
I'm not so keen on adding 6 more build configurations. It already takes 45 minutes, and I was happy when I just removed PHP 7.1's build set. I wouldn't mind *two* master configurations; 64-bit, opcache=yes, ts=1; and 32-bit, opcache=no; ts=0. To be honest, I don't think that for master we should make 32bit builds (including PHP itself) at all any more. It's 2020.